code/controllers/subsystems/emergency_shuttle.dm

```diff
@@ -225,11 +225,13 @@ var/datum/controller/subsystem/emergency_shuttle/emergency_shuttle
 
 //returns 1 if the shuttle is not idle at centcom
 /datum/controller/subsystem/emergency_shuttle/proc/online()
+	if (isnull(shuttle?.location))
+		return FALSE
```

Comment:
huh what, all you need to do is check if the shuttle is null

Reply:
kk